includes/class-cpt-curator.php

```diff
@@ -414,6 +419,67 @@ public function filter_edit_post_link( $edit_link, $post_id, $context ) {
 	}
 
 	/**
+	 * Modify the curator post trash link to read uncurate and removed the edit inline link.
+	 *
+	 * @param $actions
+	 * @param $post
+	 *
+	 * @return mixed
+	 * @since 0.1.0
+	 */	
+	public function filter_edit_trash_link( $actions, $post ) {
+		
+		if( 'cur-curator' !== $post->post_type ){
+			return;
+		}
+		
+		if ( ! current_user_can( 'delete_post', $post->ID ) ) {
+			return;
+		}
+		
+		if( ! isset( $actions['trash'] ) ) {
+			return;
+		}
+			
+		if ( EMPTY_TRASH_DAYS ) {
+			$actions['trash'] = "<a class='submitdelete' title='" . esc_attr__( 'Uncurate this item', 'cur' ) . "' href='" . get_delete_post_link( $post->ID ) . "'>" . __( 'Uncurate', 'cur' ) . "</a>";
```

Comment:
@ayebare this works, but string concatenation like this is hard to read.

I prefer to do something more along the lines of using sprintf to make the thing very readable and then fill in the variable parts after:

```
$actions['trash'] = sprintf( '<a class="submitdelete" title="%s" href="%s">%s</a>',
  esc_attr__( 'Uncurate this item', 'cur' ),
  get_delete_post_link( $post->ID ),
  esc_html__( 'Uncurate', cur' )
);
```